internal/kubernetes/endpoints.go

```diff
@@ -113,3 +121,21 @@ func (p *KubernetesProvider) updateEndpoints(eps, foreignEps []corev1.EndpointSu
 	}
 	return subsets
 }
+
+func hasToBeUpdated(home, foreign []corev1.EndpointSubset) bool {
+	if len(home) != len(foreign) {
+		return true
+	}
+	for i := 0; i < len(home); i++ {
+		if len(home[i].Addresses) > len(foreign[i].Addresses) {
+			return true
+		}
+		for j := 0; j != len(home[i].Addresses); j++ {
```

Comment:
just a curiosity: is there a reason you used `!=` instead of `<` ? (it's the same)

Reply:
I got confused, the symbols `!=` and `>` in 130 and 133 had to be swapped. Thank you!